src/sctools/test/test_metrics.py

```diff
@@ -370,5 +356,92 @@ def test_single_read_evidence(metrics, key, expected_value):
     observed = metrics[key].sum()
     assert observed == expected_value
 
-
 # todo need to add additional cell-metric specific tests (from parse_extra_fields)
+
+
+def split_metrics_file(metrics_file):
+    """
+    produces two mergeable on-disk metric files from a single file that contain the first 3/4
+    of the file in the first output and the last 3/4 of the file in the second output, such that
+    1/2 of the metrics in the two files overlap
+    """
+    with open(metrics_file, 'r') as f:
+        data = f.readlines()
+
+    header, data = data[0], data[1:]
+
+    low_split, high_split = round(len(data) * .25), round(len(data) * .75)
+    file_1, file_2 = [_test_dir + 'metrics_for_merging_%d.csv' % i for i in (1, 2)]
+
+    with open(file_1, 'w') as f:
+        f.write(header + '\n')
+        for line in data[:high_split]:
+            f.write(line + '\n')
+
+    with open(file_2, 'w') as f:
+        f.write(header + '\n')
+        for line in data[low_split:]:
+            f.write(line + '\n')
+
+    return file_1, file_2
+
+
+@pytest.fixture
+def mergeable_cell_metrics():
+    return split_metrics_file(_cell_metric_output_file)
+
+
+@pytest.fixture
+def mergeable_gene_metrics():
+    return split_metrics_file(_gene_metric_output_file)
+
+
+def test_merge_cell_metrics_cli(mergeable_cell_metrics):
+    """test the sctools merge cell metrics CLI invocation"""
+    TenXV2.merge_cell_metrics(
```

Comment:
I'm not sure I understand this test -- should it assert something?

Reply:
That's a good point. I had written the CLI tests to simply verify that they run. However, I've coded them all to return 0, so I'll verify that the correct return call is being generated. 👍